azure-pipelines.yml

```diff
@@ -61,22 +62,24 @@ jobs:
         feedsToUse: 'config'
         nugetConfigPath: '.nuget/nuget.config'
 
-    - task: VSBuild@1
+    - task: DotNetCoreCLI@2
       inputs:
-        solution: '$(solution)'
-        platform: '$(buildPlatform)'
-        configuration: '$(buildConfiguration)'
+        command: 'build'
+        projects: 'test/FunctionsV2/*.csproj'
+        arguments: '--configuration $(buildConfiguration)'
+      displayName: 'dotnet build $(buildConfiguration)'
 
     - task: VSTest@2
       inputs:
         testSelector: 'testAssemblies'
         testAssemblyVer2: |
-          **/*tests.v2.dll
+          **\\bin\\**\\*tests.v2.dll
```

Comment:
nit: here and in line 117, I see you switched from forward slashes to backslashes. I think it's better if we standardize on forward slashes for consistency.